packages/holocron/package.json

```diff
@@ -30,21 +30,21 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@americanexpress/vitruvius": "^2.0.0",
+    "@americanexpress/vitruvius": "^3.0.1",
     "hoist-non-react-statics": "^3.3.0",
     "immutable": "^4.0.0-rc.12",
     "prop-types": "^15.5.10",
-    "react-redux": "^7.1.1",
-    "redux": "^4.0.4",
     "redux-immutable": "^4.0.0",
-    "redux-thunk": "^2.1.0",
+    "redux-thunk": "^3.1.0",
     "require-from-string": "^1.2.1",
     "reselect": "^4.0.0",
     "semver": "^7.3.8",
     "shallowequal": "^1.1.0",
     "ssri": "^7.0.1"
   },
   "devDependencies": {
+    "redux": "^4.0.4",
```

Comment:
our peerdeps requires `>=5.0.1` but here we are only using `4.0.4`

Reply:
[89f4184](https://github.com/americanexpress/holocron/pull/172/commits/89f418488ef1dbea938c7d3a2710987da0a5bf15)